workflows/edit_data_packages/add_a_pre-generated_identifier.Rmd

```diff
@@ -0,0 +1,13 @@
+
+## Adding a pre generated identifier to the eml
+
+When you pre generate a UUID or DOI, the change is not automatically reflected in the "Identifier" section of the eml so this makes sure that the eml lines up with the identifier being used. 
+
+```{r, eval = FALSE}
+## Generate DOI and add to EML
+doiPid <- generateIdentifier(mn, "DOI")
+eml@packageId <- new("xml_attribute",
+                      .Data = doiPid)
+```
+
+Be sure to include the `identifier=` argument in your `publish_update` command so the pre generated identifier is applied.
```

Comment:
Make sure to cross reference. i.e. `[publish update](#publish-update)` https://bookdown.org/yihui/bookdown/cross-references.html